lambda/MAS/Views/Mail/Daily.cshtml

```diff
@@ -5,6 +5,9 @@
 @model DailyEmailViewModel
 @{
     var groupedItems = Model.Items
+        .OrderBy(g => g.Source.Title != "National Institute for Health and Care Excellence")
```

Comment:
this feels like it should be a variable as you're doing the same string comparison in the weekly.